cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -52,6 +53,26 @@ var (
 		Help:    "A histogram of latencies for indexing a repository.",
 		Buckets: prometheus.ExponentialBuckets(.1, 10, 7), // 100ms -> 27min
 	}, []string{"state"}) // state is an indexState
+
+	metricQueueLen = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "index_queue_len",
+		Help: "Current length of indexing queue by code host",
+	}, []string{"codehost"})
+
+	metricNumIndexed = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "index_num_indexed",
+		Help: "Number of indexed repos by code host",
+	}, []string{"codehost"})
```

Comment:
Will the code host really tell you much? I can't imagine it would

Reply:
i am trying to answer the question: have all repos from a code host been indexed or are there repos still needing indexing. i know this is not exact but it gives me a quick way to judge and estimate.